src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1952,7 +1856,7 @@ open_common(int (*real_open)(const char *pathname, int oflags, ...), const char
 		if (!parent && (strncmp(item_name, "/", 2) == 0))
 			rc = dfs_access(dfs_mt->dfs, NULL, NULL, X_OK | W_OK);
 		else
-			rc = dfs_access(dfs_mt->dfs, parent, item_name, X_OK | W_OK);
+			rc = dfs_access(dfs_mt->dfs, drec2obj(parent), item_name, X_OK | W_OK);
```

Comment:
we are calling drec2obj a lot of times here in this function. it's probably better to just call it once and use a temp variable for the parent dfs_obj_t handle.

Reply:
- [ ] Add temporary var parent_dfs

Reply:
Fixed with commit e283ccd3a